### ISSUE TITLE
nvda does not read the advanced network sharing options

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#8319]
### Steps to reproduce:
My control panel is configured to display the items by category.
Open the control panel.
Then open network and share.
Then open Network and Sharing Center.
Now open Change advanced sharing settings.
Inside the Advanced Sharing Settings window, walk with tab.

### Expected behavior:
Nvda should read the options:
Particular.
guest
as well as continuing to navigate with tab and there are the options:
all networks.
public networks
It should be possible to mark the options and the nvda should announce whether they are checked or not.

### Actual behavior:
nvda is muted when browsing with tab and the options:
particular.
guest.
all networks
public networks.
receive focus.

### System configuration:
NVDA version:
next 15130
NVDA Installed or portable:
instaled
Other information:
> Example: Running in a VM

Windows version:
windows 10 17134.48
Name and version of other software in use when reproducing the issue:

### Other questions:

Does the issue still occur after restarting your PC?
yes
Have you tried any other versions of NVDA?
> Please list them and the result
no

**Comment by Adriani90** [comment]
@fernando-jose-silva are you still able to reproduce this issue in äwindows 10 22 h2 update and NVDA last alpha or NVDA 2023.1?

**Comment by fernando-jose-silva** [comment]
At the moment I only have access to Windows 11 machines, and on this system it seems that these options have been transferred to the settings window, where this problem does not exist.

**Comment by Adriani90** [comment]
Given that Microsoft migrates more and more categories from the old control pannel to the new settings pannel, I think it is legimitate to close this issue as works for me due to the change in design of the settings pannel in Windows.
The control pannel will be completely removed in Windows 12.

See also this article for some background:
https://www.windowslatest.com/2022/08/01/windows-11s-next-update-wont-remove-control-panel-but-options-will-move-to-settings/
